libs/blocks/text/text.css

```diff
@@ -162,7 +167,8 @@
   justify-content: start;
 }
 
-.text-block.icon-inline .foreground [class^="body-"] {
+.text-block.icon-inline .foreground [class^="body-"],
+.text-block.icon-inline .foreground .cta-container {
```

Comment:
This is the only update from the previous PR https://github.com/adobecom/milo/pull/2098